example/lib/main.dart

```diff
@@ -18,101 +18,190 @@ class MySampleState extends State<MySample> {
   String cardHolderName = '';
   String cvvCode = '';
   bool isCvvFocused = false;
+  bool useGlassMorphism = false;
+  bool useBackgroundImage = false;
   final GlobalKey<FormState> formKey = GlobalKey<FormState>();
 
   @override
   Widget build(BuildContext context) {
+    final OutlineInputBorder border = OutlineInputBorder(
+      borderSide: BorderSide(
+        color: Colors.grey.withOpacity(0.7),
+        width: 2.0,
+      ),
+    );
     return MaterialApp(
       title: 'Flutter Credit Card View Demo',
       debugShowCheckedModeBanner: false,
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
       home: Scaffold(
-        resizeToAvoidBottomInset: true,
-        body: SafeArea(
-          child: Column(
-            children: <Widget>[
-              CreditCardWidget(
-                cardNumber: cardNumber,
-                expiryDate: expiryDate,
-                cardHolderName: cardHolderName,
-                cvvCode: cvvCode,
-                showBackView: isCvvFocused,
-                obscureCardNumber: true,
-                obscureCardCvv: true,
-                isHolderNameVisible: false,
-                onCreditCardWidgetChange: (creditCardBrand) {},
-              ),
-              Expanded(
-                child: SingleChildScrollView(
-                  child: Column(
-                    children: <Widget>[
-                      CreditCardForm(
-                        formKey: formKey,
-                        obscureCvv: true,
-                        obscureNumber: true,
-                        cardNumber: cardNumber,
-                        cvvCode: cvvCode,
-                        isHolderNameVisible: false,
-                        isCardNumberVisible: true,
-                        isExpiryDateVisible: true,
-                        cardHolderName: cardHolderName,
-                        expiryDate: expiryDate,
-                        themeColor: Colors.blue,
-                        cardNumberDecoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Number',
-                          hintText: 'XXXX XXXX XXXX XXXX',
+        resizeToAvoidBottomInset: false,
+        body: Container(
+          decoration: BoxDecoration(
+            image: !useBackgroundImage
+                ? const DecorationImage(
+                    image: ExactAssetImage('assets/bg.png'),
+                    fit: BoxFit.fill,
+                  )
+                : null,
+            color: Colors.black,
+          ),
+          child: SafeArea(
```

Comment:
Can we add `SafeArea` at top level in `body`?

Reply:
we are using background image in top of this so if safe area will be the parent of it then UI will be messed up.

Reply:
Okay got it.